README.md

```diff
@@ -183,6 +183,7 @@ Although these arguments are optional, a (\*) indicates a recommended parameter
 | Option              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                              | Default                        |
 | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------ |
 | `baseUri`           | Set the base URL of the LaunchDarkly server for this configuration. Only necessary if using a private instance of LaunchDarkly.                                                                                                                                                                                                                                                                                                                                          | `https://app.launchdarkly.com` |
+| `branch`            | The currently checked out git branch. If not provided, branch name will be auto-detected. Please provide when using CI systems that leave the repository in a detached HEAD state.                                                                                                                                                                                                                                                                                       |                                |
```

Comment:
I think we need to add support for this parameter in the circleci orb (https://github.com/launchdarkly/foundation/pull/800)

Reply:
good point. i'm gonna punt on it for now as branch detection still works fine in circle.